portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches data from servlet and sets data in comments.html. Called when
```

Comment:
I would keep this a little more high level and instead of linking to comments.html. I would describe where it sets the data. Eg.

Fetches data from servlet and set it in the comment section of portfolio site.

Also, not sure what <body> refers to?

Reply:
done